cfgov/cal/views.py

```diff
@@ -150,4 +152,4 @@ def pdf_response(request, context):
         response['Content-Disposition'] = 'attachment; filename=cfpb-leadership.pdf'
         return response
     except (urllib2.HTTPError, urllib2.URLError, BadStatusLine):
-        pass
+        return render(request, template_name, context)
```

Comment:
@chosak @vccabral that may be a good idea-- but what @kurtw has done here matches the current behavior. If we want to change that, it should be a seperate PR.